rundeckapp/grails-app/views/framework/_pluginConfigPropertyFormField.gsp

```diff
@@ -129,6 +129,17 @@
         </g:else>
 
 </g:elseif>
+<g:elseif test="${prop.renderingOptions?.(StringRenderingConstants.DISPLAY_TYPE_KEY) in [StringRenderingConstants.DisplayType.DYNAMIC_FORM, 'DYNAMIC_FORM']}">
+    <g:set var="fieldid" value="${g.rkey()}"/>
+    <g:set var="hasDynamicProperties" value="${dynamicProperties ? true: false}"/>
+    <g:set var="valueText" value="${values&&null!=values[prop.name]?values[prop.name]:prop.defaultValue}"/>
+
+    <g:hiddenField id="${fieldid}" name="${fieldname}" value="${valueText}" />
+
+    <div id="dynamic-form-vue-${fieldid}" class="dynamic-form-vue" fields="${valueText}" hasOptions="${hasDynamicProperties}"
```

Comment:
```suggestion
    <div id="dynamic-form-vue-${fieldid}" class="dynamic-form-vue" fields="${enc(attr:valueText)}" hasOptions="${hasDynamicProperties}"
```

Reply:
this is necessary because you are outputting the `<div>` as HTML directly, but for the `<g:hiddenField` and other taglib tags you don't need to encode because it is passed directly to the taglib code.